client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java

```diff
@@ -335,15 +336,9 @@ public ListView<String> getServiceList(int pageNo, int pageSize, String groupNam
         params.put(CommonParams.GROUP_NAME, groupName);
         
         if (selector != null) {
-            switch (SelectorType.valueOf(selector.getType())) {
-                case none:
-                    break;
-                case label:
-                    ExpressionSelector expressionSelector = (ExpressionSelector) selector;
-                    params.put(SELECTOR_PARAM, JacksonUtils.toJson(expressionSelector));
-                    break;
-                default:
-                    break;
+            if (SelectorType.valueOf(selector.getType()) == SelectorType.label) {
```

Comment:
should we use switch still but only combine the `none` and `default`?

Reply:
Only one condition matches，so choose to use if